src/site/content/en/blog/aem-with-workbox/index.md

```diff
@@ -0,0 +1,138 @@
+
+Please note that you can change your content or these settings at any time. When you publish your changes, the service worker will be updated at the client by the browser and a message will be presented to the user that a newer version of the PWA is available. The user can click the message to reload the application and get the latest updates. You can open the browser developer tools and applications panel to view the service worker details.
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/4trs4SyeqTJN7HQQTXst.png", alt="The DevTools service worker panel.", width="800", height="575" %}
+
+You can expand cache storage to view the content that has been cached locally
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/vNL5kbPDYti0GEkiJNBq.png", alt="The cache storage view in DevTools.", width="800", height="398" %}
+
+### The results
+
+It's time to look at the results of your hard work. With just configuration and without coding you have just enhanced your AEM site to become a Progressive Web App.
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/kKx6HOoA6z4hXoy4WZn8.gif", alt="An AEM site as a Progressive Web App.", width="364", height="598" %}
+
+Chrome developer tools provide a lighthouse audit that lets you check how compliant your web application and configuration is with Progressive Web App standards.
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/MGOAltGvFK3MCJLgFznv.png", alt="A lighthouse audit.", width="800", height="694" %}
+
+## Conclusion
+
+Progressive Web Apps provide an app-like experience for your website, that uses the cross platform and open nature of the web, at a lower cost of development and maintenance, while providing control over distribution. This enhances engagement, retention, and most importantly drives higher conversion rates. AEM in conjunction with Workbox makes it easy to enhance your existing site into a PWA with just configuration and no coding.
+
+## References
+
+* [WKND Tutorial](https://experienceleague.adobe.com/docs/experience-manager-learn/getting-started-wknd-tutorial-develop/overview.html)
+* [Service Workers](https://developers.google.com/web/fundamentals/primers/service-workers)
+* [Cache API](/cache-api-quick-guide/)
+* [Web application manifest](/add-manifest/)
+* Learn more about Adobe Experience Manager Sites: [https://business.adobe.com/products/experience-manager/sites/aem-sites](https://business.adobe.com/products/experience-manager/sites/aem-sites)
+* Learn more about Workbox: [https://developers.google.com/web/tools/workbox/guides/get-started](https://developers.google.com/web/tools/workbox/guides/get-started)
+* Workbox Caching strategies: [https://developers.google.com/web/tools/workbox/modules/workbox-strategies](https://developers.google.com/web/tools/workbox/modules/workbox-strategies)
+* Learn more about [Progressive Web Apps](/progressive-web-apps/)
```

Comment:
```suggestion
* [WKND Tutorial](https://experienceleague.adobe.com/docs/experience-manager-learn/getting-started-wknd-tutorial-develop/overview.html)
* [Service Workers](https://developers.google.com/web/fundamentals/primers/service-workers)
* [Cache API](/cache-api-quick-guide/)
* [Web application manifest](/add-manifest/)
* [Adaptive icon support in PWAs with maskable icons](/maskable-icon/)
* Learn more about Adobe Experience Manager Sites: [https://business.adobe.com/products/experience-manager/sites/aem-sites](https://business.adobe.com/products/experience-manager/sites/aem-sites)
* Learn more about Workbox: [https://developers.google.com/web/tools/workbox/guides/get-started](https://developers.google.com/web/tools/workbox/guides/get-started)
* Workbox Caching strategies: [https://developers.google.com/web/tools/workbox/modules/workbox-strategies](https://developers.google.com/web/tools/workbox/modules/workbox-strategies)
* Learn more about [Progressive Web Apps](/progressive-web-apps/)
```

Reply:
I just added a link to a web.dev article on maskable PWA icons. Thought it might be a relevant add to the resource list.